R/uniroot.all.R

```diff
@@ -0,0 +1,32 @@
+# This file was originally written by Karline Soetaert for the rootSolve package under the GPL 2. This version, modified by Nicholas Smoker, is here used by dsmodels under the terms of the GPL 2 (which dsmodels also uses).
```

Comment:
This file still needs a proper roxygen header, and description of the function it provides.  We get the following error when we try to document:
```
  Undefined global functions or variables:
    approxfun uniroot
  Consider adding
    importFrom("stats", "approxfun", "uniroot")
  to your NAMESPACE file.
```

So somehow we also need to note we're using stats and approxfun in the appropriate way.